joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstForStatementsCreator.scala

```diff
@@ -277,7 +277,7 @@ trait AstForStatementsCreator {
         resolveRequireOrLoadPath(argsAsts, callNode)
       } else if (callNode.name == "require_relative") {
         resolveRelativePath(filename, argsAsts, callNode)
-      } else if (callNode.name == "attr_accessor") {
+      } else if (callNode.name == "attr_accessor" || callNode.name == "private_class_method") {
```

Comment:
Perhaps we can create a set of these "special" symbols and just use a `.contains` check as there are likely more to come.

Reply:
Yes, this is my thinking as well. I am prepping a list based on usages I am seeing and then will do `.contains`. Lets wait a few more hours as I scour usages like this. I am also planning to add a dataflow test for this.